recipe_scrapers/tofoo.py

```diff
@@ -11,52 +12,59 @@ def host(cls):
         return "tofoo.co.uk"
 
     def author(self):
-        return "The Tofoo co."
+        raise StaticValueException(return_value="The Tofoo co.")
 
     def title(self):
-        return self.soup.find(
-            "h1", {"class": "recipe-detail__title h3 blue"}
-        ).get_text()
+        return self.soup.find("div", {"class": "hero__content"}).find("h1").get_text()
 
-    def category(self):
-        category_text = self.soup.find(
-            "div", {"class": "recipe-detail__ins h6"}
-        ).get_text()
-        normalized_category = normalize_string(category_text)
-        return normalized_category
+    def _find_hero_stat(self, label):
+        hero_stats = self.soup.find("ul", {"class": "hero__stats"})
+        if hero_stats:
+            for li in hero_stats.find_all("li"):
+                if re.search(rf"{label}:", li.get_text()):
+                    return li.get_text()
+        return None
```

Comment:
I'll investigate this further tomorrow, but generally I think it may be preferable to use a more-semantic HTML querying style to attempt to retrieve content from these pages.

Rationale: it's not clear whether `hero` is some kind of software component, design/CSS framework, third-party integration, or something else -- and as a result I think we should try to find a way to avoid referencing that (because it may be fragile, depending on what it is).

That said, maybe that'd make things significantly more difficult; I'll try experimenting with the code soon to try to figure that out.

Reply:
Ok; the `categories` and `statistics` seem challenging to precisely retrieve without using the `hero__*` CSS classname selectors.

In the case of `title`, we _could_ retrieve that from the `<title>` HTML element (and then use `str.rpartition` or similar to remove the attached brand name) - but I think that's optional; perhaps it is more consistent to use another `hero__*` classname here too.

Approving the pull request in a few moments; thanks again!